### ISSUE TITLE
Layout containing ContentBlockSimple and LinkList is missing certain CSS styles

### ISSUE THREAD

**Issue by kexuibm** [carbon-design-system/ibm-dotcom-library#1676]
In v1.5.0-rc.0 creating a layout structure that contains content block simple and link list.  We are currently using the following scss imports:

import '@carbon/grid/scss/grid.scss';
import '@carbon/layout/scss/layout.scss';
import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/layout/layout';
import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/link-list/index';
import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/index';

However the generated styles seems to be missing layout styles and link-list styles.

#### Acceptance Criteria
- [ ] Once fix is complete, add the issue creator as a reviewer to the PR and tag them with a request to confirm the fix.

**Comment by RobertaJHahn** [comment]
Raph is creating a test environment to replicate the issue.  Will talk to Ke about the findings in the meeting later today.

**Comment by RaphaelAmadeu** [comment]
Me and @kennylam met up with Ke, and proposed a workaround for this issue, as this is something Carbon side.

**Comment by ljcarot** [comment]
Is the workaround in another open issue?  Is a bug opened for Carbon?

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu Could you provide a summary of the fix and link the PR to this issue.  Will this fix be in v1.5.1 or 1.6.0?  I have moved the issue to sprint 20-06.  Thanks.  

cc:  @kennylam @ljcarot

**Comment by jeffchew** [comment]
Hi @RobertaJHahn @ljcarot , the fix is not in our codebase, but an application setup configuration which we provided them. We probably should set up a separate task to update our documentation to include that until we can align with Carbon on their end.

cc: @RaphaelAmadeu @kennylam

**Comment by RaphaelAmadeu** [comment]
The proposed work-around was creating a `.env` file and adding the following line,
`SASS_PATH=node_modules:src`,
 so SASS can correctly get the imports from the node_modules folder.

We are still investigating to figure out if this is something on our or Carbon side, and if we should open a ticket from that.

@RobertaJHahn , @kennylam , @ljcarot

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu I have created issue (#1741) to update our documentation with the work around.  The issue is included in this sprint.

cc:  @jeffchew @ljcarot

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu This issue is in the Done pipeline.  You made a comment on March 17 that you are investigating whether a fix is needed on the Carbon side or on our side.  Have you determined that? If done, can you add a comment of work has been done?  Thanks

cc:  @ljcarot
